src/odroid.md

```diff
@@ -8,18 +8,21 @@
  * Power board on|off loop
  * Odroid power cable (make sure this is plugged in to a 5V port)
  * Micro USB cable
- * USB stick (with the following code in a robot.zip from the IDE):
+ * USB stick ([with the following code in a robot.zip](https://studentrobotics.org/docs/programming/getting_code_on_the_robot)):
```

Comment:
It sort of reads this way that the link contains the code, rather than what's below.
```suggestion
 * USB stick (with the following code [in a robot.zip](https://studentrobotics.org/docs/programming/getting_code_on_the_robot)):
```